### ISSUE TITLE
"Expecting an argument" error when building

### ISSUE THREAD

**Issue by KiruthikaAdhi** [amake/flutter_charset_detector#1]
Thank you for the plugin. But it is showing this error while building

```
pub.dartlang.org\flutter_charset_detector-1.0.0\android\src\main\kotlin\com\madlonkay\flutter_charset_detector\FlutterCharsetDetectorPlugin.kt: (58, 36): Expecting an argument
```

**Comment by amake** [comment]
1. Please don't put your entire message in the subject (I've moved it to the issue body now).
2. I've never seen this, so you'll need to provide more information. Preferably a sample project that I could run to see the error.

**Comment by saviourdog** [comment]
same problem

**Comment by saviourdog** [comment]
@amake  just create new flutter2 project,add flutter_charset_detector dependencies,flutter run will occure this problem,thanks a lot

![image](https://user-images.githubusercontent.com/22761043/114259538-2e750d80-9a01-11eb-9c25-dc7f2fb6a199.png)

**Comment by amake** [comment]
OK thanks. The problem is a trailing coma in a method invocation. I will fix it very soon, but in the meantime you can work around it by upgrading your application's Kotlin version to 1.3.70 or later.

**Comment by amake** [comment]
This is fixed in 1.0.1.
